### PR TITLE
Support cli upgrade command

### PR DESCRIPTION
**Pull request by YuvalFeldman**

### CODE REVIEW

File: cmd/cli/upgrade.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	if err != nil {
+		return err
+	}
+
+	go func() {
+		for {
+			buf := make([]byte, 1024)
+			n, err := stdout.Read(buf)
+			if n > 0 {
+				fmt.Print(string(buf[:n]))
```
Comment (moshe-kabala) [inline cmd/cli/upgrade.go:64]: use `log.Info`

File: cmd/cli/upgrade.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+			}
+		}
+	}()
+
+	go func() {
+		for {
+			buf := make([]byte, 1024)
+			n, err := stderr.Read(buf)
+			if n > 0 {
+				fmt.Print(string(buf[:n]))
```
Comment (moshe-kabala) [inline cmd/cli/upgrade.go:77]: use `log.Error`

**Review by moshe-kabala — APPROVED** [review]
LGTM

File: cmd/cli/upgrade.go (hunk 1 above)
Comment (YuvalFeldman) [reply cmd/cli/upgrade.go]: Updated

File: cmd/cli/upgrade.go (hunk 2 above)
Comment (YuvalFeldman) [reply cmd/cli/upgrade.go]: Updated

**Pull request merged**